### ISSUE TITLE
Add more sources for gifs in aliucord

### ISSUE THREAD

**Issue by ItsHanibee** [Aliucord/plugin-requests#121]
### Discord Account

TheHanSolo#8785

### Plugin Description

Right now discord only pulls gifs from tenor while there's a whole lot more gif providers that do a better job, maybe a plugin where i can choose my gif providers or maybe even add my own would be pretty handy, a little 3 dot menu in the gifs section where i can toggle them or the same in the plugins page would also be fine.
If this ever becomes a real thing it would make my day ❤️

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by vy-storm** [comment]
Yes please!!  I'd personally love to see raw google image results, since gboard filters all gifs shown using a strict safe search

**Comment by mantikafasi** [comment]
https://github.com/mantikafasi/AliucordPlugins/raw/refs/heads/builds/MoreGifProviders.zip
